proto/regen/data/v1alpha1/tx.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package regen.data.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/data";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// Msg is the regen.data.v1alpha1 Msg service
+service Msg {
+
+    // AnchorData "anchors" a piece of data to the blockchain based on its secure hash,
+    // effectively providing a tamper resistant timestamp.
+    rpc AnchorData(MsgAnchorDataRequest) returns (MsgAnchorDataResponse);
+
+    // SignData allows for signing of an arbitrary piece of data on the blockchain.
+    //
+    // On-chain signatures have the following benefits:
+    // - on-chain identities can be managed using different cryptographic keys
+    //   that change over time through key rotation practices
+    // - an on-chain identity may represent an organization and through delegation
+    //   individual members may sign on behalf of the group
+    // - the blockchain transaction envelope provides built-in replay protection
+    //   and timestamping
+    //
+    // SignData implicitly calls AnchorData if the data was not already anchored.
```

Comment:
SignData has multiple `signers`, and AnchorData only one `sender`. Who will be the `sender` in this case? `signers[0]`? If yes, it would be good to clarify it here.

Reply:
We don't need a `sender` here. `sender` is just needed for `MsgRequest.GetSigners` in the other cases, it's not used otherwise. i.e. it's just needed to pay fees. Whereas `signers` is used for both `MsgRequest.GetSigners` and stored. I can improve the docs